plugins/chat/app/services/chat/lookup_user_threads.rb

```diff
@@ -84,7 +84,6 @@ def fetch_threads(guardian:)
             ::Chat::Channel
               .joins(:user_chat_channel_memberships)
               .where(user_chat_channel_memberships: { user_id: guardian.user.id, following: true })
-              .where.not("user_chat_channel_memberships.muted")
```

Comment:
This is the main part of the change, which allows us to show threads we follow from muted channels.